transcendence-app/src/auth/auth.controller.ts

```diff
@@ -93,4 +103,32 @@ export class AuthController {
   loginLocalUser(@Req() req: Request, @Body() user: LoginUserDto): User {
     return req.user;
   }
+
+  @Post('local/role')
```

Comment:
praise: I like how you implemented these endpoints

question: What do you think about moving the role routes to the authorization controller?

`local/role` -> `authorization/role`

Reply:
Yes, why not.